### ISSUE TITLE
@nuxtjs/proxy doesn't work with express

### ISSUE THREAD

**Issue by dan-gamble** [nuxt-community/modules#36]
I'm using the `nuxtjs/express` starter template and added:

```js
import proxy from 'http-proxy-middleware'
app.use('/api', proxy({ target: 'http://localhost:8000', changeOrigin: true }))
```

This works 👌

However adding:

```js
  modules: [
    '@nuxtjs/proxy'
  ],

  proxy: {
    '/api': 'http://localhost:8000'
  }
```

to `nuxt.config.js` just gives me a 404 inside nuxt. Any ideas as to why?

**Comment by pi0** [comment]
Hi! This was already mentioned in header : 

> Notice: As a limitation currently this module only works with nuxt internal server. For express,... you may have to manually register nuxt.serverMiddleware. (Will be fixed soon)

Middleware support is going to be moved from internal server to renderer so it will work with express soon. (nuxt release coming soon)

**Comment by dan-gamble** [comment]
Sorry, my bad. I should learn to read 😣

**Comment by pi0** [comment]
This is now available with all rc versions of nuxt :)

**Comment by microcipcip** [comment]
@pi0 Is there a way to configure the express server so that backpack can watch two entries at the same time?

I have the API on http://localhost:3001 and the nuxt server.js on http://localhost:3000, I am basically trying to keep them separate so that I don't have to wait 5 seconds for nuxt to rebuild every time I save the API.

My biggest headache ATM is that I cannot watch them both with a single backpack entry (I get `Error: listen EADDRINUSE`) so I had to duplicate the package.json file and backpack.config.js in the API folder. 
Now I have another problem, because when I build I could potentially avoid running the API and just import it directly on server.js

**Comment by microcipcip** [comment]
@pi0 Is there a way to configure the express server so that backpack can watch two entries at the same time?

I have the API on http://localhost:3001 and the nuxt server.js on http://localhost:3000, I am basically trying to keep them separate so that I don't have to wait 5 seconds for nuxt to rebuild every time I save the API.

My biggest headache ATM is that I cannot watch them both with a single backpack entry (I get `Error: listen EADDRINUSE`) so I have to duplicate the package.json file and backpack.config.js in the API folder. 

Basically I have the following:
- server/index.js
- api/index.js
- api/backpack.config.js
- api/package.json
- backpack.config.js
- package.json

But now I have another problem, because when I build for production I can potentially avoid running the API server and just import it directly on server.js, however now I have two duplicate package.json files that must match 100% to make this work, which looks quite cumbersome to me.

Is there any suggestion you have for this situation?
